### DISCUSSION TITLE
Support SD-XL

### DISCUSSION THREAD

**Discussion by raisen** [AbdBarho/stable-diffusion-webui-docker#541] [General]
I tried installing a SD-XL model but have been getting errors. When I researched about it, it looks like it's not yet supported by AUTOMATIC1111:

https://github.com/vladmandic/automatic/blob/dev/DIFFUSERS.md

Will I need to make any changes to the Docker setup once AUTOMATIC support it?

**Comment by AbdBarho** [comment]
Not really, once support is there it should work out of the box, I will keep an eye on any changes and update if necessary.
